src/fundus/publishers/uk/daily_mail.py

```diff
@@ -0,0 +1,36 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+    generic_topic_parsing,
+)
+
+
+class DailyMailParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = CSSSelector("#js-article-text > div:nth-child(32)")
```

Comment:
The paragraph selector doesn't work for any article I encountered so far.

Reply:
Yes, I see if as well now. I changed the css selector, it should work after I push the new version.